sklearn/preprocessing/data.py

```diff
@@ -2579,6 +2579,157 @@ def quantile_transform(X, axis=0, n_quantiles=1000,
                          " axis={}".format(axis))
 
 
+class BoxCoxTransformer(BaseEstimator, TransformerMixin):
+    """Apply the Box-Cox transform featurewise to make data more Gaussian-like.
+
+    The Box-Cox transformation is a monotonic transformation that is
+    applied to make data more Gaussian-like. This is useful for solving
+    modeling issues related to heteroscedasticity (non-constant variance),
+    or other situations where normality is desired.
+
+    Box-Cox requires input data to be strictly positive and non-zero.
```

Comment:
"strictly" means nonzero.

Reply:
I'll go with "positive and non-zero", then.

Reply:
We do use "strictly positive" (as distinct from "non-negative") elsewhere in the codebase and docs, so I'm not too worried about it being misinterpreted.

Reply:
sweet sounds good!